### ISSUE TITLE
issues with v1.0.5

### ISSUE THREAD

**Issue by ballmannweber** [fatfishdigital/zoommeeting#5]
– the plugin’s internal links like Host Meeting or Edit don’t work if the cpTrigger is set to other than admin
– while editing a meeting the zoom menu on the left looses it’s submenu
– after submitting the changed meeting there is still no submenu on the left while the main view seems to be the same
– trying to edit an entry or clicking host meeting while the submenu on the left is missing leads to a serverentry

also could you provide a very simple basic template on how to use the plugins's field on the frontend?
this doesn't seem to work:

`{% set entries = craft.entries().section('zoom').all() %}
{% for entry in entries %}
	{{ entry.zoommeetingid }}	
{% endfor %}`

**Comment by ballmannweber** [comment]
is someone listening?

**Comment by ballmannweber** [comment]
it's very disappointing that even after two weeks there is no reply and no support. unfortunately our email request was not answered either.

**Comment by fatfishdigital** [comment]
@ballmannweber We are very sorry for this delay, All your issues email were in the moved to junk folder ( dont know why). 
But will look at your issue today and get back to you soon.

**Comment by ballmannweber** [comment]
Any news regarding the bugs?

**Comment by fatfishdigital** [comment]
@ballmannweber Yes, we will be releasing this Week or probably on Monday(next week if delays), Need to do few more test this week.

**Comment by fatfishdigital** [comment]
@ballmannweber New bug fixes has been released, there still some issues that needs to be work on, which is second point of your issue sub menu disappearing on edit ( I have raised issue with craft communities as this should be easily solved by using selectedSubnavItem in twig template which is not working atm). 

for templating you can use twig template variable which is craft.zoom.get_meetingById() and passing entry.zoommeetingid[0] 
here zoommetingid is the field that you have created.


https://github.com/fatfishdigital/zoommeeting/blob/master/README.md here is the guide, if you still encounter some issue let us know.
